packages/testnet-faucets/tsconfig.json

```diff
@@ -3,5 +3,10 @@
     "compilerOptions": {
         "outDir": "lib"
     },
-    "include": ["./src/ts/**/*", "../../node_modules/web3-typescript-typings/index.d.ts"]
+    "include": [
+        "./src/ts/**/*",
+        "../../node_modules/types-bn/index.d.ts",
```

Comment:
Why are we adding this?

Reply:
Same as above comment about bn types, was getting a compilation / build error before about missing bn types but looks like it is resolved now

Reply:
So is this still needed? @BMillman19